### PR TITLE
krishna: Add assignments of QA

### PR DESCRIPTION
**Pull request by mrkrishnaupadhyay**

### CODE REVIEW

**Comment by prertik** [comment]
Reformat the PR title, it can be: `Add course materials and assignments of QA`

**Comment by mrkrishnaupadhyay** [comment]
Dai, please review the JSON assignment pull request.

File: QA Interns 2020/Assignments/Krishna Upadhyay/QA test.postman_collection.json [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+        "id": "e09d37da-75a2-4ea2-a4c2-2c2635aca2df",
+        "type": "text/javascript",
+        "exec": [
+          ""
+        ]
+      }
+    }
+  ],
+  "protocolProfileBehavior": {}
+}
```
Comment (Nischalkhadka) [inline QA Interns 2020/Assignments/Krishna Upadhyay/QA test.postman_collection.json]: Add a line space

File: QA Interns 2020/Assignments/Krishna Upadhyay/jsonAssignment.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    "errorType": "ErrorMessages",
+    "errorCode": "VALIDATION_ERROR_INVOICE_SUPPLIER_TAX_FEWER_THAN_TOTAL_OFLINE_ITEM_SUPPLIER_TAXES",
+    "groupId": null,
+    "messageKey": null,
+    "message": "Invoice Supplier Tax is less than the total of all Invoice line items Supplier Taxes",
+    "messageArguments": null,
+    "context": {}
+  }
+]
+```
```
Comment (Nischalkhadka) [inline QA Interns 2020/Assignments/Krishna Upadhyay/jsonAssignment.md]: Add a line space

**Comment by Nischalkhadka** [comment]
Do resolve the conflicts before I can merge this PR,

**Comment by mrkrishnaupadhyay** [comment]
Dai, conflict resolved!

**Pull request merged**